rero_ils/modules/documents/api.py

```diff
@@ -174,7 +174,7 @@ def index_persons(self, bulk=False):
             auth_pid = author.get('pid')
             if auth_pid:
                 from ..persons.api import Person
-                person = Person.get_record_by_pid(auth_pid)
+                person = Person.get_record_by_mef_pid(auth_pid)
```

Comment:
We have to verify the `get_record_by_mef_pid` is optimized and there are no double records in ES created.